client/src/components/ManageBookings/CurrentBooking/CurrentBookingCard.tsx

```diff
@@ -0,0 +1,32 @@
+import { Typography, Box } from '@mui/material';
+import SettingsIcon from '@mui/icons-material/Settings';
+import useStyles from './useStyles';
+
+const CurrentBookingCard = (): JSX.Element => {
+  const classes = useStyles();
+  return (
+    <Box component="div" className={classes.cardWrapper}>
+      <SettingsIcon className={classes.icon} />
+      <Typography component="div" className={classes.date}>
+        YOUR NEXT BOOKING
```

Comment:
Generally against using uppercase like this. Rather, we should set the CSS to transform the text to uppercase. Screen readers will read this out letter by letter which would be very annoying for a user with disabilities.